workflow/validate/validate.go

```diff
@@ -35,6 +35,10 @@ type ValidateOpts struct {
 	// types of executors. For example, the inability of kubelet/k8s executors to copy artifacts
 	// out of the base image layer. If unspecified, will use docker executor validation
 	ContainerRuntimeExecutor string
+
+	// IgnoreEntryPoint indicates to skip/ignore the EntryPoint validation on workflow spec.
+	// Entrypoint is optional for WorkflowTemplate and ClusterWorkflowTemplate
+	IgnoreEntryPoint bool
```

Comment:
Very minor suggestion:

```suggestion
	IgnoreEntrypoint bool
```